sources/ClangSharp.PInvokeGenerator/Cursors/Cursor.cs

```diff
@@ -82,10 +83,9 @@ protected Cursor(CXCursor handle, Cursor parent)
 
         public TranslationUnit TranslationUnit { get; }
 
-        public CXToken[] Tokenize(Cursor cursor)
+        public Span<CXToken> Tokenize(Cursor cursor)
```

Comment:
Took a dependency on `System.Memory` to avoid creating arrays and copying overhead where possible.